qa-tests-backend/scripts/migrate_protos.sh

```diff
@@ -9,4 +9,6 @@ cp -r ../proto/* ${JAVA_PATH}
 # Migrate v1 API protos from the Scanner repo
 SCANNER_DIR=$(go list -f '{{.Dir}}' -m github.com/stackrox/scanner)
 # files from gomod cache have no write permission causing problems for gradle
-cp -r --no-preserve=mode ${SCANNER_DIR}/proto/* ${JAVA_PATH}
+cp -r ${SCANNER_DIR}/proto/* ${JAVA_PATH}
+chown -R $(whoami) ${JAVA_PATH}
```

Comment:
do we need this line?

Reply:
I'm not sure. I think `cp` should create the files with the caller as owner. The ownership change is an additional safety measure.